### PR TITLE
[Merged by Bors] - Increase the size limit of the log volume

### PR DESCRIPTION
**Pull request by siegfriedweber**
# Description

Increase the size limit of the log volume.

see stackabletech/operator-rs#620

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] CRD changes approved
- [x] Helm chart can be installed and deployed operator works
- [x] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by siegfriedweber** [comment]
https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/trino-operator-it-custom/84/

File: rust/operator-binary/src/controller.rs [hunk 1]
```diff
@@ -616,12 +617,27 @@ fn build_rolegroup_config_map(
...
                 // The maximum size of all logfiles combined
                 dynamic_resolved_config.insert(
                     LOG_MAX_TOTAL_SIZE.to_string(),
-                    Some(format!("{TRINO_LOG_FILE_TOTAL_SIZE_IN_MIB}MB")),
+                    Some(format!(
+                        // Trino uses the unit "MB" for MiB.
+                        "{}MB",
+                        MAX_TRINO_LOG_FILES_SIZE
+                            .scale_to(BinaryMultiple::Mebi)
+                            .ceil()
```
Comment (sbernauer) [inline rust/operator-binary/src/controller.rs:638]: ceil or floor? :thinking:

**Review by sbernauer — APPROVED** [review]
LGTM % comment

File: rust/operator-binary/src/controller.rs (hunk 1 above)
Comment (siegfriedweber) [reply rust/operator-binary/src/controller.rs:638]: Let's assume that the developer sets a reasonable value of 10 MiB:
LOG_MAX_SIZE is 5MB
LOG_MAX_TOTAL_SIZE is 10MB
⇒ okay

If the developer sets an odd value like 2.2 MiB:
LOG_MAX_SIZE is 2MB
LOG_MAX_TOTAL_SIZE is 3MB
⇒ okay, I guess

If the developer sets a tiny value like 0.2 MiB:
LOG_MAX_SIZE is 1MB
LOG_MAX_TOTAL_SIZE is 1MB
⇒ not okay (also with floor)

**Comment by siegfriedweber** [comment]
bors merge

**Pull request closed without merging**